dvc/main.py

```diff
@@ -48,9 +47,6 @@ def main(argv=None):
 
         cmd = args.func(args)
         ret = cmd.run()
-    except LockError:
-        logger.exception("failed to lock before running a command")
-        ret = 250
```

Comment:
If this is about an error message only then why do we change the return code?

Reply:
@efiop might be able to answer this.

Reply:
@Suor What will be the command on `cmd` on Windows? `tasklist | findstr dvc`?

Reply:
LockError is based on DvcException and doesn't make it up here, as all CLI commands have `try except DvcException` in them. So we can remove this bit.